README.rst

```diff
@@ -0,0 +1,34 @@
+What is Kolibri?
+----------------
+
+Kolibri is an adaptable measured learning environment. A user can install Kolibri and serve the app on a local network,
```

Comment:
It's one role that it plays, but it certainly does more (or at least different things) to a traditional LMS.